docs/autogen.md

```diff
@@ -110,26 +112,62 @@ config_list_memgpt = [
 ]
 ```
 
+#### OpenAI example
 If you are using the OpenAI API (e.g. using `gpt-4-turbo` via your own OpenAI API account), then the `config_list` for the AutoGen agent and `config_list_memgpt` for the MemGPT AutoGen agent will look different (a lot simpler):
 ```python
 # This config is for autogen agents that are not powered by MemGPT
 config_list = [
     {
-        "model": "gpt-4-1106-preview",  # gpt-4-turbo (https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo)
+        "model": "gpt-4",
         "api_key": os.getenv("OPENAI_API_KEY"),
     }
 ]
 
 # This config is for autogen agents that powered by MemGPT
 config_list_memgpt = [
     {
-        "model": "gpt-4-1106-preview",  # gpt-4-turbo (https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo)
         "preset": DEFAULT_PRESET,
-        "model": None,
+        "model": "gpt-4",
+        "model_wrapper": None,
+        "model_endpoint_type": None,
+        "model_endpoint": None,
+        "context_window": 8192,  # gpt-4 context window
+    },
+]
+```
+
+#### Azure OpenAI example
```

Comment:
Note @sarahwooders 

In the AutoGen integration code, `config_list_memgpt` will get passed to a call to `AgentConfig`: https://github.com/cpacker/MemGPT/blob/main/memgpt/autogen/memgpt_agent.py#L51-L62

This current PR will fail because the `AgentConfig` does not take the additional params required for Azure.

However, based on the following `~/.memgpt/config` generated via a `memgpt configure` (with Azure variables set correctly), do the new variables inside of `config_list_memgpt` seem right to you at a spot check?

<img width="422" alt="image" src="https://github.com/cpacker/MemGPT/assets/5475622/2cda091a-1288-45c2-bc0c-fe9e33fc32cf">

`~/.memgpt/config`:
```
[defaults]
preset = memgpt_extras
persona = sam_pov
human = cs_phd

[model]
model = gpt-4
model_endpoint = https://XXX.openai.azure.com/
model_endpoint_type = azure
context_window = 8192

[openai]
key = XXX

[azure]
key = XXX
endpoint = https://XXX.openai.azure.com/
version = 2023-07-01-preview
embedding_deployment = text-embedding-ada-002

[embedding]
embedding_endpoint_type = azure
embedding_endpoint = text-embedding-ada-002
embedding_dim = 1536
embedding_chunk_size = 300

[archival_storage]
type = local

[version]
memgpt_version = 0.2.4

[client]
anon_clientid = ...
```